templates/patterns/gallery/gallery.ui_patterns.yml

```diff
@@ -7,34 +7,119 @@ gallery:
       label: "Items"
       description: "List of GalleryItemValueObject objects."
       preview:
-        - thumbnail:
```

Comment:
We should keep the same structure of the old pattern.

Reply:
We are keeping the same structure though, we still have a list of GalleryItemValueObjects. The objects themselves have changed, which I think its ok?